pkg/sentry/platform/kvm/bluepill_arm64.go

```diff
@@ -42,6 +42,13 @@ var (
 			sErrEsr:     _ESR_ELx_SERR_NMI,
 		},
 	}
+
+	// vcpuExtDabt is the event of ext_dabt.
+	vcpuExtDabt = kvmVcpuEvents{
```

Comment:
Note that this will need to be &kvmVcpuEvents, which will prevent the escapes: no below. The below escape error is actually a bug, since if stop the world is active, this will cause the sandbox to fail.

Reply:
I see. Thanks. I will improve it.